src/olympia/versions/models.py

```diff
@@ -92,6 +92,9 @@ class Version(OnChangeMixin, ModelBase):
     source = models.FileField(
         upload_to=source_upload_path, null=True, blank=True)
 
+    channel = models.IntegerField(choices=amo.RELEASE_CHANNELS,
```

Comment:
That's not a valid `choices`. django system checks complain about it:
`versions.Version.channel: (fields.E005) 'choices' must be an iterable containing (actual value, human readable name) tuples.`